recipes/behaviortree.cpp/all/conanfile.py

```diff
@@ -112,6 +138,20 @@ def generate(self):
 
     def build(self):
         apply_conandata_patches(self)
+        ver = Version(self.version)
+
+        path_bt_factory = ''
+        if ver < "4.0":
+            path_bt_factory = os.path.join(self.source_folder, "include", "behaviortree_cpp_v3", "bt_factory.h")
+        else:
+            path_bt_factory = os.path.join(self.source_folder, "include", "behaviortree_cpp", "bt_factory.h")
+
+        # add version tag
+        replace_in_file(self,
+            path_bt_factory,
+            "#define BT_FACTORY_H",
+            "#define BT_FACTORY_H\n#define BEHAVIORTREE_VERSION 0x%02d%02d%03d" % (int(str(ver.major)), int(str(ver.minor)), int(str(ver.patch))))
```

Comment:
The library doesn't include any version information anywhere. I wanted to provide a version to the developer. Because there are some deps which can't be used at compilation time before 4.1.1